commit 7dd56b107bd476ae57c659b6b9eb5ddd7bb26a2f (refs/changes/03/4203/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-06-13T21:01:13+00:00 (1 year, 4 months ago)
    
    OS-7019 missing break in hma_fpu_alloc
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/uts/i86pc/os/hma_fpu.c b/usr/src/uts/i86pc/os/hma_fpu.c
index f207470e85..14cfa8baed 100644
--- a/usr/src/uts/i86pc/os/hma_fpu.c
+++ b/usr/src/uts/i86pc/os/hma_fpu.c
@@ -105,6 +105,7 @@ hma_fpu_alloc(int kmflag)
 	case FP_FXSAVE:
 		bzero(fpu->hf_guest_fpu.fpu_regs.kfpu_u.kfpu_generic,
 		    sizeof (struct fxsave_state));
+		break;
 	case FP_XSAVE:
 		bzero(fpu->hf_guest_fpu.fpu_regs.kfpu_u.kfpu_generic,
 		    cpuid_get_xsave_size());
